app/Modules/Endorsement/Special/Services/SpecialEndorsementAssignment.php

```diff
@@ -22,11 +22,11 @@ public function __construct(EndorsementRequest $endorsement, User $actioner)
     public function handle()
     {
         // check if the endorsement has already been assigned.
-        if ($this->endorsementRequest->endorsement->users->contains($this->endorsementRequest->user)) {
+        if ($this->endorsementRequest->endorsement->users()->contains($this->endorsementRequest->user_id)) {
```

Comment:
Why was this change needed?

Reply:
This looks like it was a change due to change in the Auth package. It implemented a macro on the BelongsTo relationship to add a contains method, so that we don't have to retrieve all the user's from the API. Ref: https://github.com/VATSIM-UK/auth-package/blob/master/src/UKAuthServiceProvider.php#L62

Reply:
Can't really remember - I'll roll it back anyway